packages/opentelemetry-basic-tracer/package.json

```diff
@@ -3,10 +3,15 @@
   "version": "0.0.1",
   "description": "OpenTelemetry Basic Tracer",
   "main": "build/src/index.js",
+  "browser": {
+    "./src/platform/index.ts": "./src/platform/browser/index.ts",
```

Comment:
Should this point to `js` files instead? Also what happens if we only have the `"main"` entry point here?

Reply:
Seems like your suggestions fixed the problem with Core and BasicTracer for browser, will update soon, thx

Reply:
Should that fix be applied to the `core` package as well?

Reply:
yes it is fixed